docs/guides/Create-UWP-Packages-C#.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Create NuGet Packages for the Universal Windows Platform
+description: An end-to-end walkthrough of creating NuGet packages using a Windows Runtime Component for the Universal Windows Platform.
+author: karann-msft
+ms.author: karann
+ms.date: 03/21/2017
+ms.topic: tutorial
+---
+
+# Create UWP packages (C#)
+
+The [Universal Windows Platform (UWP)](https://developer.microsoft.com/windows) provides a common app platform for every device that runs Windows 10. Within this model, UWP apps can call both the WinRT APIs that are common to all devices, and also APIs (including Win32 and .NET) that are specific to the device family on which the app is running.
+
+In this walkthrough you create a NuGet package with a C# UWP component (including a XAML control) that can be used in both Managed and Native projects.
+
+## Prerequisites
+
+1. Visual Studio 2019. Install the 2019 Community edition for free from [visualstudio.com](https://www.visualstudio.com/); you can use the Professional and Enterprise editions as well.
```

Comment:
`2019` version is variable. In this [visual studio doc](https://docs.microsoft.com/en-us/visualstudio/get-started/csharp/tutorial-uwp), `If you haven't already installed Visual Studio, go to the` [`Visual Studio downloads`](https://visualstudio.microsoft.com/downloads/) `page to install it for free.` Visual Studio downloads page will always point the customer to the latest version.